src/main/java/com/powsybl/sld/server/NetworkAreaDiagramService.java

```diff
@@ -163,6 +169,13 @@ public void deleteNetworkAreaDiagramConfig(UUID nadConfigUuid) {
         nadConfigRepository.deleteById(nadConfigUuid);
     }
 
+    public String loadNetworkAreaDiagramSvgFromConfigAsync(UUID networkUuid, String variantId, UUID nadConfigUuid) {
```

Comment:
Rename method to be consistent with `generateNetworkAreaDiagramSvgAsync`

Reply:
done